src/HUDL/HUDL.cpp

```diff
@@ -3,176 +3,119 @@
  * basic echo functionality where the uart will write back whatever the user
  * enters.
  */
-#include <stdint.h>
 
-#include <EVT/io/GPIO.hpp>
-#include <EVT/io/I2C.hpp>
-#include <EVT/io/manager.hpp>
-#include <EVT/io/pin.hpp>
-#include <EVT/io/UART.hpp>
-#include <EVT/utils/time.hpp>
+// clang-format off
```

Comment:
Can you add a comment explaining why this is necessary?

Reply:
This might actually not be necessary after you fix the formatting file.

Reply:
This is just a quick band-aid fix to GitHub reformatting and causing compile errors. The issue will be researched eventually and fixed.

Reply:
Can you add a comment that says that? It would probably also make sense to make a Trello card for that to ensure that doesn't get forgotten down the line